### DISCUSSION TITLE
The future of pyspcwebgw

### DISCUSSION THREAD

**Discussion by jschlyter** [mbrrg/pyspcwebgw#19] [Announcements]
Greetings!

Together with @mbrrg, @ehn and I have transferred the code to its new home and we'll try to review all pending PRs in the near future. If you want to help with code review and testing, please let us know!

ping @ehn @crallian @CoMPaTech @fredriklj

**Comment by CoMPaTech** [comment]
Highly appreciated! ❤️

**Comment by crallian** [comment]
Wow, sounds great!
I am happy help with testing when suitable!

**Comment by CoMPaTech** [comment]
Just to be clear I kind of wrote my PR just to contribute but I didn't write it in a way to ensure 'our system' would just neatly integrate. I'll have another look at the code I'm currently running just to be sure, but there were some deviations left and right.

Backtracking;
 - My PR should still be valid (with regards to the above that it 'made it work for me' but probably needs more tinkering to work with the existing code
 - The HA system runs with below as valid configuration (2022.12.8 on supervisor 2022.12.1 (debian box, not hassos))

```
  "name": "Vanderbilt SPC",
  "documentation": "https://www.home-assistant.io/integrations/spc",
  "requirements": ["git+https://github.com/CoMPaTech/pyspcwebgw.git@flexgw#pyspcwebgw==0.6.0a1"],
  "version": "2021.1",
```

 - Double-checking the customized component I don't think there is anything other than 'time' gone by (and the link to my PR package). As I know from recent github.com/plugwise endavours there is the typing improvements, just if anyone needs to double-check; https://gist.github.com/CoMPaTech/e376e5c404b6184b8e68b903e04472c6
 - We actually do more with our spcflex gateway, but using curl to fetch status and control doors (not this module nor HA), but it would be fairly trivial to write a doorlock or switch for the appropriate spcflex calls.
